src/sortStudents.ts

```diff
@@ -1,7 +1,85 @@
 // describe Student type
 // create and export SortType enum
 // create SortOrder type
+import { SortType } from './sortTypes';
 
-export function sortStudents(students, sortBy, order) {
+type Students = {
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+};
+
+type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Students[],
+  sortBy: SortType,
+  order: SortOrder,
+): Students[] {
   // write your function
+  const copyStudents = [...students];
+
+  switch (sortBy) {
+    case SortType.Name:
+      if (order === 'asc') {
+        copyStudents.sort((prev, next) => prev.name.localeCompare(next.name));
```

Comment:
```suggestion
        copyStudents.sort((prev, next) => prev.name.localeCompare(next[sortBy]));
```
а якщо так, це скоротить кількість коду?)
Подумай у цьому напрямку